### PR TITLE
APPDEV-8529 fixing file path and adding solr restart instructions

### PR DESCRIPTION
**Pull request by sharonluong**
Small update to the readme about how to restart Solr, and a correction to the path for the `.jar` file after it's unzipped.

### CODE REVIEW

**Comment by cazzerson** [comment]
Maybe instead of a doc change, you could add a script to restart solr on each machine startup https://stackoverflow.com/a/37208655/30953

**Comment by sharonluong** [comment]
@cazzerson ready for re-review:
screen shot 2019-02-25 at 9 29 36 am

File: README.md [hunk 1]
```diff
@@ -59,6 +59,9 @@ $ mysql -u homestead jitterbug < $jitterbug-db-dump -psecret
 	5. Click Execute.
 	6. When jitterbug-items is finished indexing, repeat these steps for each core.
 
+	
+You may need to import the data to Solr again after restarting the VM, as above.
```
Comment (cazzerson) [inline README.md]: Please confirm that this is accurate.

Comment (sharonluong) [reply README.md]: removed!

**Pull request merged**